config/cloud.cfg.tmpl

```diff
@@ -99,11 +99,11 @@ cloud_init_modules:
  - set_hostname
  - update_hostname
  - update_etc_hosts
-{% if variant in ["alpine", "photon"] %}
+{% if variant in ["alpine", "mariner", "photon"] %}
```

Comment:
I see that you're basically copying whatever Photon does in this file, but does Mariner work exactly the same way? Is the `resolv_conf` module needed? Is there a reason to disable ca-certs and ssh-import-id?

Reply:
Working with my team to confirm these. will push a new commit.

Reply:
After discussion, we decided to leave those modules enabled and give the abilities to run those modules to users as long as they don't bring any side effects when no detailed configurations are specified under the module.